dotCMS/src/main/java/com/dotcms/variant/business/web/CurrentVariantSessionItem.java

```diff
@@ -0,0 +1,56 @@
+
+package com.dotcms.variant.business.web;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * Represents a session item for the current variant.
+ * This is a Flash Session Item that will be available for 10 seconds.
+ */
+public class CurrentVariantSessionItem implements Serializable {
```

Comment:
I would add a toString method that returns just the variantName.

Reply:
done https://github.com/dotCMS/core/pull/25387/commits/e19901110019d5b7017630cd3c90c930a2a91748#diff-93e02c67ffc61213dd2dab902128bbfce57006d11ccbb366f8e1452b366461d0R44